include/swift/AST/Builtins.def

```diff
@@ -718,6 +723,12 @@ BUILTIN_MISC_OPERATION(TypePtrAuthDiscriminator, "typePtrAuthDiscriminator", "n"
 //  int_instrprof_increment has type (Builtin.RawPointer, Builtin.Int64, Builtin.Int32, Builtin.Int32) -> ().
 BUILTIN_MISC_OPERATION(IntInstrprofIncrement, "int_instrprof_increment", "", Special)
 
+/// Initialize the default-actor instance in a default actor object.
+BUILTIN_MISC_OPERATION(InitializeDefaultActor, "initializeDefaultActor", "", Special)
+
+/// Destroy the default-actor instance in a default actor object.
+BUILTIN_MISC_OPERATION(DestroyDefaultActor, "destroyDefaultActor", "", Special)
```

Comment:
I think you'll need to add all of these to `visitBuiltinAddress` in `MemAccessUtils.cpp`. There's a non-exhaustive switch in there that's bitten me a few times.

Reply:
@atrick Andy, can you clarify whether it's necessary/correct to update MemAccessUtils.cpp?  It looks like it's visiting uses of addresses.  The default-actor state is part of the class instance, but it's not otherwise modeled in SIL at all.

Reply:
@rjmccall  It looks like you added builtins that take a reference operand. Those will look like any other reference-escaping operation. MemAccessUtils won't try to build an access path for those, but they will look like unknown uses of the object that may overlap with other access to the object. If you want to make sure alias analysis disambiguates those from property access, then you can file a performance radar for that general problem. `AccessPathDefUseTraversal::visitSingleValueUser` will need to be updated, but so will MemBehavior etc.

Reply:
Okay, it's probably not high-priority, but filed as rdar://72170139.